src/reducers.js

```diff
@@ -87,6 +89,68 @@ export function Reducer(prevState = {}, action) {
   if (!newState.pathQueries)
     newState.pathQueries = [];
 
+  // update url after state change unless on redux initialization or
+  // explicitly bypassed
+  if (action.updateUrl === true && prevState !== undefined)
+    updateUrl(newState);
+
+  // update document title after state change
+  updateTitle(newState);
```

Comment:
I suppose this could be done in components since, as it is right now in this PR, it is always called in the reducer. 

As a side note of explanation: The only reason I've moved `updateUrl` to the reducer is because I require the flag/parameter `action.updateUrl`. As far as I know, there's no good way to pass a flag through a react component prop change/update.

Reply:
I'm going to move this function to the main app component, since that seems like the most logical place to put it. Using `react-helmet` seems like overkill just to do `document.title = `. Ultimately, this app is going to be placed inside a larger page that will already have a `<head>` section defined, so there's also no need to use the other features of `react-helmet` like `<meta>` or `<link>`.